### DISCUSSION TITLE
Viseron issues in CasaOS with Reolink cameras

### DISCUSSION THREAD

**Discussion by udbrky** [roflcoopter/viseron#751] [General]
Came here from Reddit. I'm having a new issue now and tried a reinstall and still not able to get Viseron to run. It's grayed out in CasaOS. My CasaOS address is 10.69.0.14 and I'm using port 8888, but it won't start up.

Yesterday, I had an issue with starting the Stream.

I installed using this:

docker run --rm \
  -v {recordings path}:/recordings \
  -v {config path}:/config \
  -v /etc/localtime:/etc/localtime:ro \
  -p 8888:8888 \
  --name viseron \
  --device /dev/dri \
  roflcoopter/viseron:latest

Here are the settings I used:

![settings](https://github.com/roflcoopter/viseron/assets/170052788/020742e8-457b-4600-a72d-cc4e7dc36239)

**Comment by roflcoopter** [comment]
Unfamiliar with CasaOS, but can you check the docker logs? That should give you an indication to why it is not staring

**Reply by udbrky** [reply]
Sure can when I get home. I looked around for the logs but didn't find them in Casa's appdata folders.

**Comment by mikikiv** [comment]
This issue happened to me with Casaos 0.4.11 - downgrading CasaOS to 0.4.9 and 'reseting' the Viseron config file fixed this. The issue is the config file text was being duplicated whenever I opened it, which is a bug in that version of CasaOS

**Reply by udbrky** [reply]
Thanks for that info. I just installed on docker and it's been fine. Is there a setting to auto run in docker?

**Reply by mikikiv** [reply]
Not that I'm aware of. Check the Viseron config file for duplicated text and fix it, then install viseron using the same settings in your screenshot should get it open again; if that's actually this issue you were facing
